challenges-webapp/src/main/webapp/vue-app/challenges/components/AddChallenge.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <div class="border-box-sizing clickable addChallengeButton">
+    <v-btn class="btn btn-primary">
+      <i class="fas fa-plus pr-1"></i>
+      <span class="ms-2 d-none d-lg-inline">
+        {{ $t('exoplatform.challenges.button.addChallenge') }}
```

Comment:
It's better to always use simple keys for labels:
exoplatform.challenges.button.addChallenge = > challenges.button.addChallenge